src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPanelMapping.java

```diff
@@ -57,12 +57,48 @@ public static List<String> getAuthorisationSubTypes() {
         return Collections.emptyList();
     }
 
-    public static List<PanelPreference> getPanelPreferences() {
-        // TODO Adjournments - loop to go through Judicial members that are need to be included or excluded
-        // TODO Potentially used with Manual overrides
-        //      Will need Judicial Staff Reference Data
+    public static List<PanelPreference> getPanelPreferences(SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
+        Adjournment adjournment = caseData.getAdjournment();
+        ArrayList<PanelPreference> panelPreferences = new ArrayList<>();
 
-        return new ArrayList<>();
+        if (!referenceDataServiceHolder.isAdjournmentFlagEnabled()
+            || isNoOrNull(adjournment.getAdjournmentInProgress())) {
+            return panelPreferences;
+        }
+
+        if (nonNull(adjournment.getPanelMember1())) {
+            PanelPreference medicallyQualifiedPanelMember =  PanelPreference.builder()
+                .memberID(adjournment.getPanelMember1().getIdamId())
+                .memberType("PANEL_MEMBER")
+                .requirementType(RequirementType.OPTIONAL_INCLUDE)
+                .build();
+            panelPreferences.add(medicallyQualifiedPanelMember);
```

Comment:
DRY. create a separate function that returns a `panelReference` and add that to the `panelReferences` to remove redundancy. 
You can also add another lvl of abstraction and create `fetchPanelReferences` and check for all the nonNulls there add to local list and return it

Reply:
Made the necessary review comment changes and committed.